tools/wego-app-dev.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
```

Comment:
```suggestion
---
apiVersion: v1
kind: Namespace
metadata:
  name: wego-system
---
apiVersion: apps/v1
```

I got the error 
```
STEP 5/5 — Deploying
     Applying YAML to cluster
     Tried to apply objects to cluster:
       → wego-app:deployment

Build Failed: namespaces "wego-system" not found
```

when running `kind create cluster && make api-dev`

Reply:
okay this then runs into a bunch of other stuff, e.g. `[K8s EVENT: ReplicaSet wego-app-7747ff49cf (ns: wego-system)] Error creating: pods "wego-app-7747ff49cf-" is forbidden: error looking up service account wego-system/wego-app-service-account: serviceaccount "wego-app-service-account" not found
`

is it possible to just `wego install` and then replace the container?

Reply:
@aclevername Yeah that would have to be the steps. Install first to get all the other K8s resources, then you can start developing. Do I need to updated the README?

Reply:
@jpellizzari Definitely. :)

Reply:
Or even better would be if Tilt would do that for you, since it's local setup. :)